Hw1/Task1.py

```diff
@@ -0,0 +1,12 @@
+# Задача 1:  Найдите сумму цифр трехзначного числа.
+
+a = int(input("Введите трёхзначное число: "))
+print("Вы ввели: ", a)
+
+x = a // 100
+y = a % 10
+b = a // 10 
+z = b % 10
```

Comment:
Можно одну строчку убрать.
```suggestion
x = a // 100
y = a % 10
z = a // 10 % 10
```